docs/docs/templates/ui-components.md

```diff
@@ -0,0 +1,76 @@
+---
+description: *Add a component description of 54 characters maximum*
+keywords:
+  - UI library
+  - ecommerce
+  - react
+---
+
+import PropsSection from "@site/src/components/PropsSection/PropsSection";
+
+*Add ComponentName* is *add component meaning* used to *add the component function*.
+
+## Import
+
+```tsx
+import { ComponentName } from '@faststore/ui'
+```
+## Usage
+
+```tsx live
+ <ComponentName> </ComponentName>
+```
+### Variants
+
+> *Use this section only if the component has variants.*
+#### VariantName
+
+> Add a variant example
+
+```tsx live
+<ComponentName>
+    Add the variant example
+</ComponentName>
```

Comment:
```suggestion
```tsx live
/* Replace ComponentName and VariantName accordingly. */
<ComponentName
  variant="variantName"
/>
```
```